decidim-elections/app/models/decidim/elections/election.rb

```diff
@@ -78,6 +78,20 @@ def minimum_answers?
         questions.all? { |question| question.answers.size > 1 }
       end
 
+      # # Public: Checks if the election results are published and election finished
+      # #
+      # # Returns a boolean.
+      def results_published?
+        bb_status == "results_published"
+      end
+
+      # # Public: Checks if the election results present
+      # #
+      # # Returns a boolean.
+      def results?
+        bb_status == "results"
+      end
```

Comment:
Not super important but beware of the double comment 😄

Reply:
Yeah. 😅